quil-py/quil/program/__init__.pyi

```diff
@@ -330,7 +329,9 @@ class CalibrationSet:
         function returns the previous calibration. Otherwise, None is returned.
         """
         ...
-    def insert_measurement_calibration(self, calibration: MeasureCalibrationDefinition) -> Optional[MeasureCalibrationDefinition]:
+    def insert_measurement_calibration(
+        self, calibration: MeasureCalibrationDefinition
+    ) -> Optional[MeasureCalibrationDefinition]:
```

Comment:
Would it be quick to cover these stubs with a CI lint so no one merges misformatted code like I did here?

Reply:
I've been wanting to move to `ruff` for all our projects, so I went ahead and did that, addressed the new issues it flagged, and added lints and formatting to the default checks.